services/registry/server.js

```diff
@@ -31,6 +31,19 @@ const myMiddles = [
   require('./middleware/object-store')
 ];
 
-const server = boltzmann.make(router, myMiddles);
-server.listen(process.env.PORT, '0.0.0.0');
-logger.info(`listening on port: ${process.env.PORT}`);
+const main = () => {
```

Comment:
This is just a little modification to make closing the server a little more graceful in Docker.